packages/aws-cdk-lib/aws-dynamodb/README.md

```diff
@@ -631,6 +631,15 @@ const table = new dynamodb.TableV2(this, 'Table', {
 });
 ```
 
+`recoveryPeriodInDays` specifies the number of preceding days for which continuous backups are taken and maintained. If this property is set without configuring `pointInTimeRecovery` then point-in-time recovery will be enabled by default.
```

Comment:
Happy to discuss default behaviour documented here if there's any concerns. IMO it makes sense to set `pointInTimeRecovery` = `true` by default if `recoveryPeriodInDays` is set (with the ability to override `pointInTimeRecovery` = `false` if desired).